### ISSUE TITLE
preload is broken in 1.0.6

### ISSUE THREAD

**Issue by jfgodoy** [pinojs/pino-debug#14]
the preload feature is broken in version 1.0.6. To check, you can run this simple test

`index.js:`
```
const debug = require('debug')('test');
debug('hello');
```

In `1.0.5`
```
$ DEBUG=* node -r pino-debug index.js 
{"level":20,"time":1513633535846,"msg":"hello","pid":14734,"hostname":"xxx","ns":"test","v":1}
```
In `1.0.6` nothing is printed

**Comment by mcollina** [comment]
Ouch, did you find the root cause? Would you mind sending a PR?

**Comment by jfgodoy** [comment]
I did a long research...
In the `pinoDebug` function, the variable `namespaces` starts as an empty array. Opts is empty too, so at the end, the `debug.enable(namespaces.join(','))` is equivalent to `debug.enable('')`. The debug library, save this value in `process.env.DEBUG`, so nothing is printed.

**Comment by lukaszewczak** [comment]
Oh my, this is my fault, I'm sorry @mcollina , I had correct version in #11.

**Comment by mcollina** [comment]
Fixed in 1.0.7.

**Comment by jfgodoy** [comment]
great! thanks @mcollina
